libraries/AP_HAL_ChibiOS/hwdef/CUAV-7-Nano/README.md

```diff
@@ -0,0 +1,80 @@
+# CUAV-7-Nano Flight Controller
+
+The CUAV-7-Nano flight controller produced by [CUAV](https://www.cuav.net).
+
+## Features
+
+- STM32H753 microcontroller
+- 2 IMUs: IIM42652 and BMI088
+- builtin IST8310 magnetometer
+- 2 barometers: BMP581 and ICP20100
+- microSD card slot
+- USB-TypeC port
+- 1 ETH network interface
+- 5 UARTs plus USB
+- 14 PWM outputs
+- 3 I2C ports
+- 3 CAN ports (two of which share a CAN bus and one is an independent CAN bus)
+- Analog RSSI input
+- 3.3V/5V configurable PWM ouput voltage
+
+## Pinout
+
+![CUAV-7-Nano_interface_definition.png](CUAV-7-Nano-pinout.png)
+
+## UART Mapping
+
+- SERIAL0 -> USB
+- SERIAL1 -> UART7 (TELEM1)
+- SERIAL2 -> UART5 (TELEM2)
+- SERIAL3 -> USART1 (GPS&SAFETY)
+- SERIAL4 -> UART8 (GPS2)
+- SERIAL7 -> USART3 (FMU DEBUG)
+
+The TELEM1 and TELEM2 ports have RTS/CTS pins, the other UARTs do not have RTS/CTS.
+
+## RC Input
+
+RC input is configured on the RCIN pin, at one end of the servo rail, marked RCIN in the above diagram. 
+
+## PWM Output
+
+The CUAV-7-Nano flight controller supports up to 14 PWM outputs.
+
+The 14 PWM outputs are in 6 groups:
+
+- PWM 1-4 in group1 (TIM5)
+- PWM 5 and 6 in group2 (TIM4)
+- PWM 7 and 8 in group3 (TIM1)
+- PWM 9, 10 and 11 in group4 (TIM8)
+- PWM 12 in group5 (TIM15)
+- PWM 13 and 14 in group6 (TIM12)
+
+Channels within the same group need to use the same output rate. If any channel in a group uses DShot then all channels in the group need to use DShot.
+
+First first 8 PWM outputs of CUAV-7-Nano flight controller support switching between 3.3V voltage and 5V voltage output. It can be switched by parameter configuration.
```

Comment:
```suggestion
First first 8 PWM outputs of CUAV-7-Nano flight controller support switching between 3.3V voltage and 5V voltage output. It can be switched to 5.5V by setting GPIO 80 high by setting up a Relay to control it.
```

Reply:
still need this change